docs/components/base/_index.md

```diff
@@ -152,9 +152,9 @@ Turn the base in place, rotating it to the given angle (degrees) at the given an
 **Parameters:**
 
 - `angle` [(float)](https://docs.python.org/3/library/functions.html#float): The angle to spin in degrees.
-Negative implies backwards.
+Positive implies forwards.
```

Comment:
what does "foward" mean when spinning? It's not a head-over-heels somersault, so...?

Reply:
Ooh you're so right. Would it be positive = turning left?
```suggestion
Positive implies turning to the left.
```

Reply:
This wording came from the Python docs (so those also need to be changed when we figure this out--I'll follow up), and the Go docs don't specify either.

Reply:
I *think* this is a safe assumption because https://github.com/viamrobotics/rdk/blob/main/components/base/wheeled/wheeled_base.go#L254 spin tells the left motors to go negative RPM